### PR TITLE
Enable Utility feature

### PR DESCRIPTION
**Pull request by seethatgo**

### CODE REVIEW

File: util/config.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+config1:abc
+
+config2:123
```
Comment (ademotest) [inline util/config.txt]: This config value should be 567. The old value was 123.

File: util/feature.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+feature 1 implementation
+
+feature 2 implementation
+
+feature 3 implementation
```
Comment (ademotest) [inline util/feature.txt:6]: Let's add a bit more to the feature 2 implementation. I think it should handle multiple widgits.

**Review by ademotest — CHANGES_REQUESTED** [review]
Please fix the config values and add the implementation to feature2

File: util/config.txt (hunk 1 above)
Comment (seethatgo) [reply util/config.txt]: @ademotest see the latest commit for the change

File: util/feature.txt (hunk 2 above)
Comment (seethatgo) [reply util/feature.txt:6]: @ademotest see the latest commit for the implementation

**Comment by seethatgo** [comment]
Modified code based on review

**Review by ademotest — APPROVED** [review]

**Pull request merged**